hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/security/x509/SecurityConfig.java

```diff
@@ -228,6 +223,44 @@ public SecurityConfig(ConfigurationSource configuration) {
         TimeUnit.MILLISECONDS);
   }
 
+  /**
+   * Check for certificate validity configuration.
+   */
+  private void validateCertificateValidityConfig() {
+    if (maxCertDuration.isNegative() || maxCertDuration.isZero()) {
+      LOG.error("Certificate maxDuration {} should not be zero or negative",
```

Comment:
Please add the property name in the error message, like this 

String msg = "Property " + HDDS_X509_MAX_DURATION + " should not be zero or negative"; 
Log.error(msg);
throw new IllegalArgumentException(msg);

Reply:
Updated.